src/components/ModalVersion.ts

```diff
@@ -0,0 +1,16 @@
+import template from './ModalVersion.html';
+import * as ko from 'knockout';
+import client from '../api/client';
+
+class ModalVersion {
+  version: ko.Observable<string | null>;
+
+  constructor() {
+    client.get('/version').then(function (response) {
```

Comment:
@DianaVargasOrellana, what happens if we need to get the version in another place? should we use the URL path '/version' too, right? what happens if the path changes? we will need to change the strings in all those places. So, we have a problem here. If we remember the "fundamental theorem of software engineering", we should add a level of indirection. What should be that level of indirection?

Reply:
I have added a new variable to store this address